containers/Dockerfile.buildenv

```diff
@@ -177,14 +176,20 @@ RUN ln -s $(which clang++-5.0) /usr/local/bin/clang++ \
     && ln -s $(which clang-5.0) /usr/local/bin/clang \
     && ln -s $(which clang-format-5.0) /usr/local/bin/clang-format \
     && ln -s $(which clang-tidy-5.0) /usr/local/bin/clang-tidy
-RUN git clone https://github.com/UIUC-PPL/charm \
+# Build charm with both GCC and clang. We check out only a specific branch
+# in order to reduce the repo size, We also remove a bunch of temporary
+# files from the repo in order to reduce the disk usage. We do not build
+# with debug symbols, and build with O2 to reduce build size.
```

Comment:
In the above,
- you have a `, We` that should either be a period or a lower-case `w`.
- I'm not clear if by "temporary files" you're referring to the clone without history, or the removal of the docs and examples. If the former, I'd suggest adjusting the previous clause to "only a specific branch (without any history)", if the later, I'd suggest saying "unused" files rather than "temporary", because the docs/examples aren't temporary....